.travis.yml

```diff
@@ -4,12 +4,13 @@ python:
   - "2.7"
   - "3.3"
   - "3.4"
+  - "3.5.0"
```

Comment:
You can just use "3.5" here, technically no need to fix the regex. Still like your version better though.